CooperationHullMainSite/Views/Shared/_footer.cshtml

```diff
@@ -1,10 +1,24 @@
-﻿        <div class="row">
-            <div class="col-lg-6 col-md-12">
-                <h3>Cooperation Hull 2024</h3>
+﻿<footer class="homepage-block p-3">
+    <div class="footer-content">
+        <div class="footer-content-left">
+            <p class="heading-md">Cooperation Hull ©️ 2024</p>
+            <div class="footer-links" role="navigation">
+                <a asp-area="" asp-controller="Home" asp-action="PrivacyPolicy">Privacy</a>
+                <a asp-area="" asp-controller="Home" asp-action="ContactUs">Contact Us</a>
+                <a asp-area="" asp-controller="Home" asp-action="CookiePolicy">Cookies Policy</a>
             </div>
-        <div class="col-lg-6 col-md-12 footer-links" role="navigation" aria-label="Footer">
-                <p> <a asp-area="" asp-controller="Home" asp-action="PrivacyPolicy">Privacy</a></p>
-                <p> <a asp-area="" asp-controller="Home" asp-action="ContactUs">Contact Us</a></p>
-                <p> <a asp-area="" asp-controller="Home" asp-action="CookiePolicy">Cookies Policy</a></p>
-            </div>
-        </div>
\ No newline at end of file
+        </div>
+        <div class="footer-socials">
+            <a asp-area="" asp-controller="Home" asp-action="Facebook">
+                <img src="./assets/img/icon-fb.svg" />
```

Comment:
Suggestion only
I've got font-awsome added to the solution, could possible use that instead of adding the files.

Reply:
Nice! I think I'll keep this here for now, for full parity with the designs and since it's already working. But happy to re-evaulate a bit later if FA proves better